docs/quickstart/index.md

```diff
@@ -0,0 +1,324 @@
+                
+            ```sh
+            pmm-admin config --server-insecure-tls --server-url=https://admin:admin@X.X.X.X:443
+            ```
+
+    3. Add the MySQL database using Performance Schema:  
+
+        ```sh 
+        pmm-admin add mysql --query-source=perfschema --username=pmm --password=<your_password> MYSQL_SERVICE
+        ```
+    ??? info "Alternative database connection workflows"
+        While the default instructions above focus on connecting a Self-hosted MySQL database, PMM offers the flexibility to connect to various MySQL databases, including [AWS RDS](../setting-up/client/aws.md), [Azure MySQL](../setting-up/client/azure.md) or [Google Cloud MySQL](../setting-up/client/google.md). 
+
+        The PMM Client installation also comes with options: in addition to the installation via Package Manager described above, you can also install it as a Docker container or as a binary package. Explore [alternative PMM Client installation options](../setting-up/client/index.html#binary-package) for more information.
+
+        Additionally, if direct access to the database node isn't available, opt for the [PMM Client installation via User Interface](../setting-up/client/mysql.html#with-the-user-interface) instead. 
+
+=== "PostgreSQL"
+
+    To connect a PostgreSQL database: 
+    { .power-number}
+
+    1. Create a PMM-specific user for monitoring:
+        
+        ```
+        CREATE USER pmm WITH SUPERUSER ENCRYPTED PASSWORD '<your_password>';
+        ```
+
+    2. Set up the `pg_stat_monitor` database extension and configure your database server accordingly. This default extension is developed by Percona and mirrors PostgreSQL's `pg_stat_statements` extension with added functionalities such as bucket-based data aggregation, enhanced accuracy, and the ability to expose query examples. 
+    
+        If you need to use the `pg_stat_statements` extension instead, see [Adding a PostgreSQL database](../setting-up/client/postgresql.md) and the [`pg_stat_monitor` online documentation](https://docs.percona.com/pg-stat-monitor/configuration.html) for details about available parameters.
+
+    3. Set or change the value for `shared_preload_library` in your `postgresql.conf` file:
+
+        ```ini
+        shared_preload_libraries = 'pg_stat_monitor'
+        ```
+
+    4. Set up configuration values in your `postgresql.conf` file:
+
+        ```ini
+        pg_stat_monitor.pgsm_query_max_len = 2048
+        ```
+
+    5. Install PMM Client via Package Manager on the database node to optimize server-side resources:
+         
+        1. Install Percona Release Tool:
+
+            ```sh
+            wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+            dpkg -i percona-release_latest.generic_all.deb
+            ```
+        2. Install the PMM Client package:
+            
+            ```sh
+            apt update
+            apt install -y pmm2-client
+            ```
```

Comment:
these command work only for debian-based OSes and we should mention red-hat based command also.